packages/cli/src/commands/generate/dbAuth/templates/forgotPassword.tsx.template

```diff
@@ -16,9 +16,9 @@ const ForgotPasswordPage = () => {
     }
   }, [isAuthenticated])
 
-  const usernameRef = useRef<HTMLInputElement>(null)
+  const ${usernameCamelCase}Ref = useRef<HTMLInputElement>(null)
   useEffect(() => {
-    usernameRef?.current?.focus()
+    ${usernameCamelCase}Ref?.current?.focus()
   }, [])
 
   const onSubmit = async (data: { username: string }) => {
```

Comment:
I think we could do `${usernameCamelCase}` here, right? (And of course also when we pass it to `forgotPassword`

Reply:
Yeah I totally agree. I've implemented changes along these lines in d79294ab836098e8e9d2ff0c3328329db0159f56.